media-gfx/pqiv/pqiv-2.12.ebuild

```
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit edo linux-info toolchain-funcs xdg-utils
```

Please add blank line after `EAPI=`:

```suggestion
EAPI=8

inherit edo linux-info toolchain-funcs xdg-utils
```